### PR TITLE
go get: add messages redirect

### PR DESCRIPTION
**Pull request by derekperkins**

### CODE REVIEW

**Comment by alainjobart** [comment]
DCO! :)

**Comment by derekperkins** [comment]
Yeah, I did that via the GitHub UI. I'm redoing the commit locally and will force push soon.

**Comment by derekperkins** [comment]
DCO complete

**Comment by derekperkins** [comment]
DCO complete

File: vitess.io/go-import/vitess.html [hunk 1]
```diff
@@ -1,4 +1,6 @@
 <html><head>
 <meta name="go-import" content="vitess.io/vitess git https://github.com/vitessio/vitess">
 <meta name="go-source" content="vitess.io/vitess     https://github.com/vitessio/vitess https://github.com/vitessio/vitess/tree/master{/dir} https://github.com/vitessio/vitess/blob/master{/dir}/{file}#L{line}">
+<meta name="go-import" content="vitess.io/messages git https://github.com/vitessio/messages">
```
Comment (enisoc) [inline vitess.io/go-import/vitess.html]: The process for adding a new redirect is more like:

* copy `go-import/vitess.html` to `go-import/messages.html` and update the URLs
* edit `_redirects` and copy the line related to `/vitess/*` and change it to refer to `/messages/*`

**Comment by derekperkins** [comment]
@enisoc Thanks for the guidance. It should be ready now.

**Comment by enisoc** [comment]
Netlify preview looks correct. Merging.

**Pull request merged**